docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -868,6 +868,44 @@ Behavior for
   * Functions `toStartOfDay`, `toStartOfHour`, `toStartOfFifteenMinutes`, `toStartOfTenMinutes`, `toStartOfFiveMinutes`, `toStartOfMinute`, `timeSlot` return `DateTime` if their argument is a `Date` or `DateTime`, and they return `DateTime64` if their argument is a `Date32` or `DateTime64`.
 :::
 
+## toUnixTimestampEx
```

Comment:
Name `toUnixTimestampEx` doesn't look really good because of that `Ex`. `Ex` means "extended", it's kind of vague and it's almost always better to be more precise in naming.

Doesn't `toUnixTimestampEx(str, [timezone])` return the same as `toUnixTimestamp64Seconds(toDateTime64(str, 0, [timezone]))`?

Maybe it's better to extend `toUnitTimestamp64Seconds` to allow its arguments to be `(str, [timezone])`?

Reply:
Yes, `toUnixTimestampEx(str, [timezone]) `return the same as `toUnixTimestamp64Seconds(toDateTime64(str, 0, [timezone]))`. OK, this way may be better.

Reply:
@vitlibar Delete `toUnixTimestampEx` and just add `toUnixTimestamp64Seconds`, so we can use funcion composation to like `toUnixTimestamp64Seconds(toDateTime64(str, timezone)) `to support negative value if date is before 00:00:00 UTC on Thursday, 1 January 1970.